src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/LayerDescriptor.java

```diff
@@ -1,19 +1,19 @@
 package au.org.emii.geoserver.wms;
 
 public class LayerDescriptor {
-    public final String layer;
+    public final String groupLayerName;
```

Comment:
layer: geoserver layer
variable: variable on the ncwms side

Either way, this can be done in a different piece of work.